swift/apple/FirezoneNetworkExtension/Adapter.swift

```diff
@@ -489,10 +419,83 @@ class Adapter: @unchecked Sendable {
     }
   }
 
+  /// Handles network path updates from NWPathMonitor.
```

Comment:
Do we necessarily have to re-order this? It makes it a bit difficult to diff, what is actually changing.